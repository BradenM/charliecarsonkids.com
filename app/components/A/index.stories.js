/**
 *
 * Stories for A
 *
 *
 */

import { storiesOf } from '@storybook/react';
import React from 'react';
import A from '.';

storiesOf('A', module).add('default', () => (
  <A href="https://www.google.com">A Link</A>
));
