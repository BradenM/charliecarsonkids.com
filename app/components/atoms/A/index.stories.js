/**
 *
 * Stories for A
 *
 *
 */

import React from 'react';
import A from '.';

export default {
  title: 'Design|Atoms/A',
  component: A,
};

export const defaultLink = () => <A href="https://www.google.com">A Link</A>;
