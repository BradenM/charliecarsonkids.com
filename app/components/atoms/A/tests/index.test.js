/**
 *
 * Tests for A
 *
 *
 */

import React from 'react';
import { render } from '@testing-library/react';
import A from '../index';

describe('<A />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(<A />);
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(<A />);
    expect(firstChild).toMatchSnapshot();
  });
});
