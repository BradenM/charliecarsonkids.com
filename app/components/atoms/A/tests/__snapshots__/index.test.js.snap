// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<A /> Should render and match the snapshot 1`] = `
<a
  class="A-ml9y9l-0 gDIWOl"
/>
`;
